packages/web-app-preview/src/App.vue

```diff
@@ -64,14 +64,21 @@
           @toggle-next="next"
         />
       </div>
-    </template>
+      <file-side-bar :open="sideBarOpen" :active-panel="sideBarActivePanel" :space="space" />
```

Comment:
Hence you already rename vars, I'd  love to rename this to isSideBarOpen

Reply:
done 👍